AlphaWallet/Info.plist

```diff
@@ -2,6 +2,11 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>App Transport Security Settings</key>
```

Comment:
The key/value is invalid here. But you can remove them because it's already included at https://github.com/AlphaWallet/alpha-wallet-ios/blob/894d99f4ca06b9e95573fac41b47b2d71f99fd80/AlphaWallet/Info.plist#L81